sdcm/nemesis.py

```diff
@@ -1435,7 +1435,9 @@ def disrupt_no_corrupt_repair(self):
         # prepare test tables and fill test data
         for i in range(10):
             self.log.debug('Prepare test tables if they do not exist')
-            self._prepare_test_table(ks=f'drop_table_during_repair_ks_{i}', table='standard1')
+            self._prepare_test_table(ks=f'drop_table_during_repair_ks_{i}', table='standard1', pre_create_objects=True)
+
+            self.cluster.wait_for_schema_agreement()
```

Comment:
you can drop this, as you are already doing it inside `_prepare_test_table`

Reply:
Don't we need to wait for schema agreement after we processed the previous keyspace (populated the table with data) and before we are going to create a new one?

Reply:
you call `wait_for_schema_agreement` already twice inside `_prepare_test_table`
so you don't have to call it again in here, but confirm that in the flow in `_prepare_test_table`

Reply:
As agreed, the only thing we need in this workaround is to wait for the schema agreement in `disrupt_no_corrupt_repair`.
I kept it here, but removed it from `_prepare_test_table`.